app/project_type/project_type.py

```diff
@@ -71,30 +71,19 @@ def _get_config_contents(self):
             config_contents = f.read()
         return config_contents
 
-    def fetch_project(self, executors=None, project_type_params=None):
+    def fetch_project(self):
         """
         Fetch the project onto the local machine.
 
         Runs once per machine per build. Runs the project_type's retrieve command (fetch, reset, etc) and produces
         a list of per-executor project_types.
-
```

Comment:
Interesting that you removed this stuff now. Though we have no use for this right now, we will have to add this kind of functionality once we add the per-executor setup and teardown.